tsconfig.json

```diff
@@ -2,13 +2,14 @@
   "compilerOptions": {
     "target": "es2015",
     "module": "commonjs",
-    "declaration": true,
+    // "declaration": true,
```

Comment:
Why? Needed because the _findOrCreate_ plugin is now included here?

Reply:
Yes, that's correct